framework/src/postprocessors/FindValueOnLine.C

```diff
@@ -24,7 +24,7 @@ InputParameters validParams<FindValueOnLine>()
   params.addParam<Point>("start_point", "Start point of the sampling line.");
   params.addParam<Point>("end_point", "End point of the sampling line.");
   params.addParam<Real>("target", "Target value to locate.");
-  params.addParam<unsigned int>("depth", 30, "Maximum number of bisections to perform.");
+  params.addParam<unsigned int>("depth", 36, "Maximum number of bisections to perform.");
```

Comment:
Let's think about this a bot more. there is a clear maximum number of iterations after which cutting an interval of two double precision numbers just _cannot_ give you a value that is different from both of the interval bounds. We might be able to just drop this parameter.